filter_plugins/config_encoders.py

```diff
@@ -147,6 +147,8 @@ def encode_apache(
                                 quote_all_nums=quote_all_nums,
                                 quote_all_strings=quote_all_strings,
                                 block_type='value')
+                        else:
+                            rv = rv.rstrip()
```

Comment:
I would prefer to handle the space where it's created instead of just removing it at the end. What about to use something like this (from the line 138)?

```python
                        rv += "%s<%s" % (indent * level, s['name'])

                        if 'operator' in s:
                            rv += " %s" % s['operator']

                        if 'param' in s:
                            rv += ''
                            rv += encode_apache(
                                s['param'],
                                convert_bools=convert_bools,
                                convert_nums=convert_nums,
                                indent=indent,
                                level=level+1,
                                quote_all_nums=quote_all_nums,
                                quote_all_strings=quote_all_strings,
                                block_type='value')
```